.stylelintrc

```diff
@@ -8,13 +8,13 @@
     "annotation-no-unknown": [
       true,
       {
-        "ignoreAnnotations": ["default"],
+        "ignoreAnnotations": ["default"]
       }
     ],
     "at-rule-no-unknown": null,
     "media-query-no-invalid": null,
     "no-descending-specificity": null,
-    "unit-allowed-list": ["rem", "em", "%", "ms", "s", "deg", "vh", "vw", "fr"],
+    "unit-allowed-list": ["rem", "em", "%", "ms", "s", "deg", "vh", "vw", "fr", "px"],
```

Comment:
I wonder if it would be better to remove it from here and add an inline `stylelint ignore` for the block of lines.